plugins/clock/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+project(clock)
```

Comment:
```suggestion
```

Reply:
@traversaro we can remove this because it is a low level plugin that is not referenced in other CMakeLists?

Reply:
Not really. The main point is that `project` only make sense inside the root CMakeLists.txt of a project, and typically it contain project-specific metadata like the version and the programming languages used (see https://cmake.org/cmake/help/latest/command/project.html). In some specific cases, it could make sense to have it in a subdirectory, but in general it is only useless code that can create confusion.